docs/manual/java/guide/cluster/PersistentEntityRDBMS.md

```diff
@@ -33,11 +33,12 @@ Lagom uses the [`akka-persistence-jdbc`](https://github.com/dnvriend/akka-persis
 
 We advise against using H2 in production, however, it is suitable for use in development and testing.
 
+
 In Lagom's default configuration, Lagom will use Play's JDBC support to configure and create a connection pool. Details on how to configure it can be found [here](https://www.playframework.com/documentation/2.6.x/JavaDatabase). Play should be configured to provide a JNDI binding for the datasource, by default Lagom binds it to `DefaultDS`.
 
-Lagom then configures `akka-persistence-jdbc` to use that `DefaultDS` JNDI binding. `akka-persistence-jdbc` uses [Slick](http://slick.lightbend.com/) to map tables and manage asynchronous execution of JDBC calls. This means we need to configure it to use the right Slick profile for your database, by default Lagom will use the H2 profile.
+Lagom then configures a [Slick](http://slick.lightbend.com/) Database to use that datasource in combination with a [AsyncExecutor](http://slick.lightbend.com/doc/3.2.1/api/index.html#slick.util.AsyncExecutor) that manages the thread pool for asynchronous execution of Database I/O Actions. Lagom will also take care that the connection pool is configured correctly according to the AsyncExecutor settings. The Slick Database is then bound to the JNDI name `DefaultDB` and it's used to configure the `akka-persistence-jdbc` plugin.
 
-So for example, to configure a PostgreSQL database, you can add the following to your `application.conf`:
+The `akka-persistence-jdbc` plugin uses Slick to map tables and manage asynchronous execution of JDBC calls. This means we need to configure it to use the right Slick profile for your database.
```

Comment:
I was a bit confused by the use of `we`. IDK what we do in other parts of the docs but I'd rephrase into " This means Lagom internally configures the `akka-persistence-jdbc` plugin to use the right Slick profile..." . Would that make sense?

Reply:
Sure, will change it.